subprojects/core/src/main/java/org/gradle/initialization/StartParameterBuildOptions.java

```diff
@@ -63,8 +63,8 @@
         options.add(new BuildCacheOption());
         options.add(new BuildCacheDebugLoggingOption());
         options.add(new WatchFileSystemOption());
-        options.add(new WatchFileSystemVerboseLoggingOption());
```

Comment:
You need to add `VerboseVfsLoggingOption` as well, right?

Reply:
Where did that go?

Reply:
OK, re-added.